rest/src/test/java/org/envirocar/server/rest/util/OSMTileRendererTests.java

```diff
@@ -29,4 +34,65 @@ public void SaveImageWorks() throws IOException {
 			osmt.saveImage(new BufferedImage(768, 512, BufferedImage.TYPE_INT_RGB), "1235456");
 	}
 	
+	@Test
```

Comment:
could you explain (via code comments) why the above two methods `WMTServiceWorks` and `SaveImageWorks` expect an `IOException`? Also, follow the guidelines for class methods: `lowerCamelCase`.